booster_bdd/features/stackAnalyses.feature

```diff
@@ -0,0 +1,9 @@
+@osio.regular
+Feature: Verify Stack Analysis Report in OpenShift.io
+
+  Background:
+    Given I am logged in to OpenShift.io
+
+  Scenario: Check that the stack-analyses returns a valid response for maven ecosystem
```

Comment:
Please tag the scenario by the `@osio.zabbix-metric.stack-analyses` tag so that the result can be pushed into zabbix along with the rest.
Thanks

Reply:
+1